script/citest.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 if [ ! -d _build ]; then
 	echo "no _build dir found; you should run cibuild.sh first"
-	exit 1
+	exit 0
```

Comment:
I need some more explanation here - why does this fail the coverity build?  Again, this seems like something that might could use an explicit flag that the coverity build can pass in so that we don't stop running our build+tests because of some subtle change.

Reply:
All of the three new "build" steps (cibuild, citest & cileaks) will be run in order now, hence the need to "fail" successfully those 2 new steps so we don't. Alternatively, I could make the coverity.sh explicit instead of calling that from cibuild.sh…